config-example.yml

```diff
@@ -0,0 +1,7 @@
+host: localhost
+port: 8080
+
+consumerKey: PLACEHOLDER_CONSUMER_KEY
```

Comment:
give more context for the keys so it's more explicit what they are used for.  use a hierachy